### DISCUSSION TITLE
How to make current version as default when homepage loaded?

### DISCUSSION THREAD

**Discussion by TheBestOrNothing** [facebook/docusaurus#4873] [Q&A]
Dear all , The docusaurus is very useful for me to build my personal website. Now the v1 have been released, and will maintain it for a long time before thinking about v2 (current version). After the homepage loaded, the v1 is selected defaultly. Could docusaurus let the v2 load defaultly when users access the homepage? Thanks.

**Comment by slorber** [comment]
`lastVersion: 'current'` in docs plugin config?

https://docusaurus.io/docs/versioning#figure-out-the-behavior-for-the-current-version

**Comment by Siemienik** [comment]
As far I good understand your situation, you need to set `routeBasePath` to `/` for the documentation plugin config (or docs section of theme config)
```js
        /**
         * URL route for the docs section of your site.
         * *DO NOT* include a trailing slash.
         * INFO: It is possible to set just `/` for shipping docs without base path.
         */
        routeBasePath: '/',
```

Source: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs

**Comment by TheBestOrNothing** [comment]
@Siemienik Thanks for your information.  My purpose is to load unreleased version (like 2.0.0-beta.1 🚧 in docusaurus ) by default when the homepage opened.  After change configuration with your guide, we can not build successfully. In my project, The  **docs** folder is in the outside of website folder, so the **path** is assigned to '../docs' as showed in the following. What configuration should adapt to let unreleased version loaded by default?

```
      '@docusaurus/preset-classic',
      {
        docs: {
          showLastUpdateAuthor: false,
          showLastUpdateTime: true,
          editUrl:
            'https://github.com/vulnsystem/www.gitcoins.io/blob/documentation/website/',
          path: '../docs',
          sidebarPath: require.resolve('./sidebars.json'),
          remarkPlugins: [require('@react-native-website/remark-snackplayer')],
          editCurrentVersion: true,
          onlyIncludeVersions:
            process.env.PREVIEW_DEPLOY === 'true'
              ? ['current', ...versions.slice(0, 2)]
              : undefined,
        },
```

**Comment by slorber** [comment]
> After change configuration with your guide, we can not build successfully.

We can't help you if you don't tell why the project refuse to build, it should

I suggest using `lastVersion: 'current'` in your configuration

**Reply by TheBestOrNothing** [reply]
The project build failed when use lastVersion: 'current'. The configuration and failed build informaiton you can find from [latest action](https://github.com/vulnsystem/www.gitcoins.io/actions) of my repository.
1. [The configuation changed commit](https://github.com/vulnsystem/www.gitcoins.io/commit/e57525499252e9d652a478771e0a1dc55ccb5886)
2. [Build detailed information](https://github.com/vulnsystem/www.gitcoins.io/runs/2802349405?check_suite_focus=true)

**Reply by slorber** [reply]
Read the log carefully. You now have broken links that you need to fix. You can also deploy your site with broken links if you really want to by disabling this security
